src/Symfony/Component/Semaphore/Store/Resources/redis_put_off_expiration.lua

```diff
@@ -9,10 +9,12 @@ if added == 1 then
 end
 
 -- Extend the TTL
-local curentTtl = redis.call("TTL", weightKey)
-if curentTtl < now + ttlInSecond then
-    redis.call("EXPIRE", weightKey, curentTtl + 10)
-    redis.call("EXPIRE", timeKey, curentTtl + 10)
```

Comment:
should have been the new desired TRL here.

Reply:
Got it. But even if you solution has a smaller complexity, I prefer the new one which is more accurate